README.md

```diff
@@ -282,6 +284,13 @@ const config = {
 };
 ```
 
+### startTransition
+
+If your version of React Native doesn't support `startTransition`, you can provide fallback implementation:
+```jsx
+  <FlagProvider startTransition={fn => fn()} ></FlagProvider>
+```
```

Comment:
I think this should be a react native usage section, not a startTransition section.

Reply:
It is in the React Native section and startTransition subsection.